jobs/kola-aws.Jenkinsfile

```
@@ -73,6 +73,23 @@ try { timeout(time: 90, unit: 'MINUTES') {
                     --aws-credentials-file=\${AWS_FCOS_KOLA_BOT_CONFIG}/config \
                     --aws-region=us-east-1""")
 
+        if (params.ARCH == "x86_64") {
+            stage('Xen') {
+                def tests = params.KOLA_TESTS
+                if (tests == "") {
+                    tests = "basic"
+                }
+                fcosKola(cosaDir: env.WORKSPACE,
+                         build: params.VERSION, arch: params.ARCH,
+                         extraArgs: tests,
+                         skipUpgrade: true,
+                         skipBasicScenarios: true,
```

```suggestion
                         skipBasicScenarios: false,
```

We just want it to run basic scenarios, right? Is there a way through coreos-ci-lib to get it to just run basic scenarios?